### PR TITLE
Adds an explanation for each profile

### PR DESCRIPTION
**Pull request by Annamarie2019**
My apologies if this is a duplicate. (I had some confusion when I forked a previous version, but I don't see the PR anywhere and the article is not updated, so I'm thinking I need to resubmit correctly.) 
I think I get the process for subsequent forking and PRs, so they should go more smoothly from here on out.

### CODE REVIEW

File: docs/how-to/default-profile.md [hunk 1]
```diff
@@ -31,15 +31,20 @@ If you previously configured your IPFS node to use another profile, let's say `b
...
-- `flatfs`
-- `badgerds`
-- `server`
-- `randomports`
-- `default-datastore`
-- `local-discovery`
-- `test`
-- `default-networking`
-- `lowpower`
+### Available only when initializing the node:
```
Comment (johnnymatthews) [inline docs/how-to/default-profile.md:34]:
```suggestion
### Available only when initializing the node

These profiles can only be used prior to running `ipfs init`.
```

Try not to add punctuation to titles. I can cause issues with markdown links.

File: docs/how-to/default-profile.md [hunk 2]
```diff
@@ -31,15 +31,20 @@ If you previously configured your IPFS node to use another profile, let's say `b
...
+- `flatfs` - the default and most tested datastore. Stores each block as a separate file. Use when you want a simple and reliable datastore, need garbage collection on a small datastore (<= 10GiB), or you're concerned about memory.
+- `badgerds`- the fastest datastore. Use when performance is critical. Will not properly reclaim space when your datastore is smaller than several gigabytes. Uses up to several gigabytes of memory.
+
+### Available at any time:
```
Comment (johnnymatthews) [inline docs/how-to/default-profile.md:38]:
```suggestion
### Available at any time

These profiles can be used anytime after running `ipfs init`. You may have to restart your IPFS node for the profile changes to take effect.
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Suggested a couple of changes.

**Comment by Annamarie2019** [comment]
Closing this PR:
- found better solution
- found better PR workflow
Issue #604 now resolved in  https://github.com/ipfs/ipfs-docs/pull/1024

**Pull request closed without merging**